indico/modules/events/operations.py

```diff
@@ -184,7 +184,7 @@ def clone_event(event, n_occurrence, start_dt, cloners, category=None, refresh_u
         'end_dt': end_dt,
         'timezone': event.timezone,
         'title': event.title,
-        'description': event.description,
+        'description': event.description or '',
```

Comment:
@ThiefMaster, this fixes the issue with the empty description section but I'm not sure if this is the best place to fix it. Calling event.description when the description is empty returns `RichMarkup('')` and assigning that to the new event causes RichMarkup to add the extra preformatted wrapper `RichMarkup('<div class="preformatted"></div>')`

Reply:
i think you could also just use `event._description` there to avoid triggering any special  RichMarkup logic

Reply:
i recommend also cloning the `render_mode` in that case. right now it's always HTML but we have a PR to support markdown there and then it'll be important to preserve both the original description and the render mode

Reply:
In the RenderModeMixin the default is`return cls.default_render_mode` is returned when there is only one possible option so nothing is being stored.